app/res/values-v11/themes.xml

```diff
@@ -81,6 +81,8 @@
         <item name="android:paddingLeft">@dimen/title_paddingLeft</item>
 	</style>
     
-    
+    <style name="Theme.Dialog" parent="@android:style/Theme.Holo.Light.Dialog">

```

Comment:
@ctsims Looked into this a little more, I don't think it actually has a global effect, it's only if you set an activity's style to Theme.Dialog. But I also can't really figure out what this is meant to be doing / why it's here, so would be fine with removing?